app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/ViewReturnController.scala

```diff
@@ -73,7 +74,7 @@ class ViewReturnController @Inject() (
               subscribedDetails,
               sentReturn.representativeType,
               sentReturn.isIndirectDisposal,
-              None
+              Some(isFirstReturn)
```

Comment:
shouldn't this be `Some(!isFirstReturn)` - the template is expecting `isFurtherReturn`

Reply:
actually, it should be fixed in the BE - I think I did have that at one point but when I changed the implementation y'day to what we discussed I forgot to add it in again. Fixed in BE now.

Reply:
the template is expecting `isFurtherReturn` though so shouldn't it still be `!isFirstReturn` here?

Reply:
if you look at what you wrote in CPDP-2473:
> return flag to FE to say whether or not a first return is being returned
I flip the boolean value to reflect whether it is a first return or not.

If you think that this should negated here then it really doesn't make sense to have a field called `isFirstReturn` - it should just be `isFurtherReturn`.

Reply:
it's more accurate to return `isFirstReturn` from the BE when we display a return since when we do amends, the value of that flag will be the same for further returns and amended returns. We're treating amended returns in a similar way to further returns but they're not the same thing.

Reply:
OK. I'll flip the logic in FE. Very confusing design :man_shrugging: